com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass.md

```diff
@@ -98,6 +98,7 @@ In this snippet, we fetch a lot of useful input data that you might need in your
 | **Sampling the camera color with lods is only available in after and before post process passes**. Calling `CustomPassSampleCameraColor` at before rendering will only return black. |
 | **DrawRenderers Pass chained with FullScreen Pass**: In multi-pass setups where you draw objects in the camera color buffer and then read it from a fullscreen custom pass, you'll not see the objects you've drawn in the passes before your fullscreen pass (unless you are in Before Transparent). |
 | **MSAA**: When dealing with MSAA, you must check that the `Fetch color buffer` boolean is correctly setup because it will determine whether or not you'll be able to fetch the color buffer in this pass or not. |
+| **Before Pre-Refraction and After post-process**: on these injection points the camera color buffer set as target for the fullscreen pass is the same as the one you can sample from in the shader. It means that you can't directly sample the camera color buffer if you render to the camera color buffer because of Read/Write restrictions on certain platforms. You'll have to split your effect in two passes using the custom color buffer as an intermediate buffer to avoid reading and writing simultaneously to the same buffer. |
```

Comment:
"from in the shader" does not sound right to me, although I'm not a native English speaker. Maybe @JordanL8 has a better suggestion?

Reply:
On these injection points, the camera color buffer set as the target for the fullscreen pass is the same as the one you can access inside the shader. Because the camera color buffer is the target, and because of read/write restrictions on certain platforms, you cannot directly sample from the camera color buffer inside the shader. Instead, you need to split your effect into two passes and use the custom color buffer as an intermediate buffer. This avoids reading and writing simultaneously to the same buffer.